### PR TITLE
Added uploader periodic job and uploader plugin readme

### PR DESCRIPTION
**Pull request by fgimenez**
Signed-off-by: Federico Gimenez <federico.gimenez@gmail.com>

### CODE REVIEW

**Comment by fgimenez** [comment]
/hold

Testing with the branch proposed on #686, I'll revert this to master once all is working.

File: github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml
Comment (rmohr) [inline github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml:294]: I don't think that we have bazelisk installed in the builder image (the image contains bazel only at the moment). Running `go get -u github.com/bazelbuild/bazelisk` may solve that.

**Review by rmohr — CHANGES_REQUESTED** [review]
Once we are sure that bazelisk is present, I will create a job. Looks great.

File: github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml
Comment (fgimenez) [reply github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml]: Thx @rmohr following our online conversation i've pushed changes to use the bazel version present in the image, which is the required one.

Comment (fgimenez) [reply github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml]: Finally the bazel version installed in the image doesn't match the required, pushed changes to install it as suggested

File: github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml
Comment (rmohr) [inline github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml:296]: That is a nice idea. in general we have golang already in there.  The issue is that the environment is not initialized properly, since go was installed via gimme.

File: github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml
Comment (rmohr) [inline github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml:293]: I think that if you put `/entrypoint.sh` here (https://github.com/kubevirt/kubevirt/blob/master/hack/builder/Dockerfile#L109) you should have a working env.

Comment (fgimenez) [reply github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml:293]: Thanks a lot for the pointer, the problem is that `/entrypoint.sh` uses `eval` and currently the generic PR script relies on `eval` too,  using `/entrypoint.sh` leads to nested evals and escaping issues. 

Instead I've pushed changes to download the bazelisk binary (no need for go in the path), I've also fixed some paths that were wrong bc of the different orgs for each repo (this should be removed once the generic PR script is merged), PTAL.

File: github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml
Comment (rmohr) [inline github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml:298]: Why is `-p` added here? Are you sure that this is needed?` We would also need `-dry-run=false`.

**Review by rmohr — CHANGES_REQUESTED** [review]

File: github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml
Comment (rmohr) [reply github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml:298]: Ah never mind regarding the `-p`. I thought for a moment that it was passed to `uploader`, which it is not.

Comment (fgimenez) [reply github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml]: yep, added `-dry-run=false`, moved to project-infra periodics and added containerized-data-importer as extra_ref plus a call to execute the uploader on it

**Pull request merged**
